frontend/javascripts/libs/stl_exporter.js

```diff
@@ -0,0 +1,159 @@
+// @noflow
```

Comment:
This code was copied (nearly) as is, right? Maybe add a link to the original source, this way we could also check for newer versions if there are bugs or if we need additional information :)

Reply:
Good point, I added a link!